Frontend/ui-customer/src/Screens/TicketBookingFlow/PlaceDetails/View.purs

```diff
@@ -1283,4 +1288,13 @@ getFilteredSlots slots state =
       if currentDate == (convertUTCtoISC state.data.dateOfVisit "YYYY-MM-DD") then do
         let currentTime = convertUTCtoISC (getCurrentUTC "") "HH:mm:ss"
         filter (\slot -> (convertUTCTimeToISTTimeinHHMMSS slot.slot) > currentTime) slots
-      else slots
\ No newline at end of file
+      else slots
+
+checkIfSameDayBookingNotAllowedForToday :: ST.TicketBookingScreenState -> Boolean
+checkIfSameDayBookingNotAllowedForToday state = 
+      let isTodayDate = (getCurrentDatev2 "") == state.data.dateOfVisit
+          allowSameDayBooking = case state.data.placeInfo of
+            Just (TicketPlaceResp placeInfo) -> fromMaybe true placeInfo.allowSameDayBooking
+            Nothing -> true
+      in
```

Comment:
can we make this and `isSameDayBookingAllowed ` function common as they are used at multiple places ?